Userland/Services/WindowServer/Compositor.h

```diff
@@ -110,6 +111,9 @@ class Compositor final : public Core::Object {
     OwnPtr<Gfx::Painter> m_front_painter;
     OwnPtr<Gfx::Painter> m_temp_painter;
 
+    RefPtr<Gfx::Bitmap> m_active_window_bitmap;
+    OwnPtr<Gfx::Painter> m_active_window_painter;
```

Comment:
Hmm, I don't think we need to keep these images around here in WindowServer.

Instead, let's create them on the fly and have `bitmap_of_active_window()` return a `RefPtr<Gfx::Bitmap>`.